docs/tokenomics.md

```diff
@@ -5,12 +5,45 @@ title: HDX Tokenomics
 
 import useBaseUrl from '@docusaurus/useBaseUrl'; 
 
-## Supply Breakdown
+## Purpose
+
+The HDX token is a governance token that allows staked token holders to decide the future of the protocol. Our mission with the HydraDX DAO is to distribute all decision-making to create a trustless liquidity protocol built around community-growth and self-sustainability. 
+
+To have a vote in the HydraDX DAO, and to contribute to the determination of any of the topics raised by community members, one must hold the HDX governance token. For more specifically on the governance process, please read our [Democracy](https://docs.hydradx.io/democracy_intro) documentation.
+
+HDX will initially be used for the following:
+
+- Setting the base network swap fee (the user may change this to any asset)
+- Voting on protocol upgrades
+- Voting on topics raised by community members (inclusive of allocation of Protocol-Owned Liquidity)
+
+## HDX Token Allocation
+
+Upon the launch of the HydraDX DAO, 10 billion HDX tokens were minted. However through the [governance-approved supply reduction](https://hydradx.subsquare.io/democracy/referendum/7), the total supply has been reduced to 6.5 billion HDX tokens.
+
+The allocation of these tokens is as follows:
+
+- LBP Participants - 30.5%
+- Founders and team - 12.5%
+- Investors - 10.6%
+- HDX Crowdloan - 7.6%
+- BSX Crowdloan - 1.9%
+- DAO treasury - 5.5%
+- Collators - 3.9%
+- Growth - 27.6%
+  
 <div style={{textAlign: 'center'}}>
   <img src={useBaseUrl('/tokenomics/supply_breakdown.jpg')} />
 </div>
 
-## Supply Vestings
+## HDX Emission Schedule
+
+As of Sept 2023, ~2.6 billion of HDX tokens are in circulation.
+
 <div style={{textAlign: 'center'}}>
   <img src={useBaseUrl('/tokenomics/supply_vestings.jpg')} />
 </div>
+
+There is currently no concrete emission/release schedule for HDX tokens residing in the Treasury and Growth allocations. HydraDX intends to distribute the supply of HDX in the treasury and growth funds to help fund ecosystem development where opportunities may arise. All of these distributions will be completely discussed transparently to the community beforehand, utilizing snapshot votes to probe sentiment.
+
+Token distributions range from a variety of developmental purposes, (eg. liquidity provider rewards, integrations/partnerships with other projects, etc).
```

Comment:
It's not just developmental purposes - maybe also specify growth. And I would mention bonds here alongside LP.